.github/workflows/sync-to-aur.yml

```diff
@@ -0,0 +1,45 @@
+name: Sync package to AUR
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save ssh key
+        run: |
+          mkdir -p ~/.ssh
+          echo ${{ secrets.AUR_SSH_KEY }} > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+
+      - name: Install and configure git
+        run: |
+          sudo apt-get install git
+          git config --global user.email ${{ secrets.GIT_COMMIT_EMAIL }}
+          git config --global user.name ${{ secrets.GIT_COMMIT_NAME }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
What is this for? We don't use submodules.

Reply:
See https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

We can use 1 here because there will be no `HEAD~1` then.
So it can be 0 or any number greater than 1.

Reply:
Oh, I thought it was about submobules, not history. Does using `2` instead of `0` improve fetching speed?

Reply:
I think so. Using 2 now.